tracer/src/Datadog.Trace/ClrProfiler/README.md

```diff
@@ -0,0 +1,119 @@
+﻿## Automatic Instrumentation
+
+The _ClrProfiler_ folder contains the majority of code required for automatic instrumentation of target methods. Since v2.0.0, we exclusively use "Call Target" modification, in which we rewrite the target method to add our instrumentation. 
+
+### Creating a new automatic instrumentation implementation
+
+Creating a new instrumentation implementation typically uses the following process:
+
+1. Identify the operation of interest that we want to measure
+2. Find an appropriate instrumentation point in the target library. You may need to use multiple instrumentation points, and you may need to use different targets for different _versions_ of the library
```

Comment:
```suggestion
2. Find an appropriate instrumentation point in the target library. You may need to use multiple instrumentation points (as for instance we do not support yet interfaces), and you may need to use different targets for different _versions_ of the library
```

Reply:
Well, we _do_ support that now? 😛

Reply:
Yes, for abstract classes, no for interfaces. (It will cost more to support that in native side)